Tasks/FtpUploadV1/task.json

```diff
@@ -18,8 +18,8 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 165,
```

Comment:
Looks like there are changes not related to tests migration

Reply:
Thank you! Resolved by merging task migration PR